src/containers/Storage/VDiskPopup/VDiskPopup.tsx

```diff
@@ -6,16 +6,37 @@ import {Label, Popup, PopupProps} from '@gravity-ui/uikit';
 import {InfoViewer, InfoViewerItem} from '../../../components/InfoViewer';
 
 import {EFlag} from '../../../types/api/enums';
-import {TVDiskStateInfo} from '../../../types/api/vdisk';
+import type {TVDiskStateInfo} from '../../../types/api/vdisk';
 import {stringifyVdiskId} from '../../../utils';
 import {bytesToGB, bytesToSpeed} from '../../../utils/utils';
+import {isFullVDiksData} from '../../../utils/storage';
+
+import type {IUnavailableDonor} from '../utils/types';
 
 import {NodesHosts, preparePDiskData} from '../PDiskPopup';
 
 import './VDiskPopup.scss';
 
 const b = cn('vdisk-storage-popup');
 
+const prepareUnavailableVDiskData = (data: IUnavailableDonor, poolName?: string) => {
+    const {NodeId, PDiskId, VSlotId} = data;
+
+    const vdiskData: InfoViewerItem[] = [{label: 'State', value: 'not available'}];
+
+    if (poolName) {
+        vdiskData.push({label: 'StoragePool', value: poolName});
+    }
+
+    vdiskData.push(
+        {label: 'NodeId', value: NodeId ?? '-'},
+        {label: 'PDiskId', value: PDiskId ?? '-'},
+        {label: 'VSlotId', value: VSlotId ?? '-'},
```

Comment:
my toxic trait is to point out that hyphen `-` is not dash `—`, and dash is better for marking unavailability